src/components/interstitials/interstitials.tsx

```diff
@@ -0,0 +1,52 @@
+import { type FC } from "react";
+import { SplashScreen } from "~/components/index.js";
+import { LetsBustSomeBots, StartChatting } from "~/assets/icons/index.js";
+import { Avatar, Stack } from "@mui/material";
+import {
+  CHARACTERS,
+  CHAT,
+  getCharacterSplashScreen,
+  getCharacterTitle,
+  VOTING,
+} from "~/constants/index.js";
+import { type PlayerType } from "~/types/index.js";
+import { theme } from "~/styles/theme.js";
+
+import { styles } from "./styles.js";
+
+interface Props {
+  splashScreenVariant?: typeof CHAT | typeof VOTING;
+  localPlayer: PlayerType;
+}
+
+export const Interstitials: FC<Props> = ({
```

Comment:
Isn't "interstitial" related to the usage of ads?

Reply:
Interstitial is another way to say splash screen. Got it from the designs